libs/designer-ui/src/lib/displayText/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from '@fluentui/react';
+
+export interface DisplayTextEditorProps {
+  displayTextText: string;
+  relativeLinkText?: string;
+  relativeLink?: string;
+  openRelativeLink?: (relativeLink: string) => void;
+}
+
+export const DisplayTextEditor: React.FC<DisplayTextEditorProps> = ({
+  displayTextText,
+  relativeLinkText,
+  relativeLink,
+  openRelativeLink,
+}): JSX.Element => {
+  return (
+    <div className="msla-displayText-editor-container">
+      {displayTextText ?? ''}
```

Comment:
NIT: is the `displayTextText` ever going to be null or undefined? What I mean is that the `nullish` operator (??) is never going to fall to the empty string case, since according to props is going to be there always